app/workers/trello_monitor.rb

```diff
@@ -0,0 +1,38 @@
+require 'bot/trello_helper'
+
+class TrelloMonitor
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+  include MiqToolsServices::SidekiqWorkerMixin
+  sidekiq_options :queue => :trello_bot, :retry => false
```

Comment:
Oh, right ... I changed that in case I wanted to run it and not accidentally run the other workers...

I'm not altogether clear on how to do that, but I thought this might be an approach?